homeassistant/components/advantage_air/binary_sensor.py

```diff
@@ -83,7 +87,7 @@ class AdvantageAirZoneMyZone(AdvantageAirEntity, BinarySensorEntity):
     _attr_entity_registry_enabled_default = False
     _attr_entity_category = EntityCategory.DIAGNOSTIC
 
-    def __init__(self, instance, ac_key, zone_key):
+    def __init__(self, instance: advantage_air, ac_key: int, zone_key: str) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3